lib/HotModuleReplacementPlugin.js

```diff
@@ -82,23 +84,81 @@ class HotModuleReplacementPlugin {
 			}
 		);
 
-		const addParserPlugins = (parser, parserOptions) => {
+		const createAcceptHandler = (parser, paramDependency) => {
```

Comment:
```suggestion
		const createAcceptHandler = (parser, ParamDependency) => {
```

Make it clear that this is a class